inc/theme-hooks.php

```diff
@@ -123,6 +123,53 @@ function alcatraz_output_sub_page_nav() {
 	}
 }
 
+add_action( 'alcatraz_entry_header_inside', 'alcatraz_output_default_entry_header' );
+/**
+ * Output the default entry header inner content.
+ *
+ * @since  1.0.0
+ *
+ * @param  int  $post_id  The current post ID.
+ */
+function alcatraz_output_default_entry_header( $post_id = 0 ) {
+
+	if ( ! $post_id ) {
+		$post_id = get_the_ID();
+	}
+
+	alcatraz_the_entry_title( $post_id );
+	alcatraz_the_entry_meta( $post_id );
+}
+
+add_action( 'alcatraz_entry_footer_inside', 'alcatraz_output_default_entry_footer' );
```

Comment:
This still feels a bit messy to me, and wasn't quite what I was expecting. For example, I was trying to customize the footer for one of my child themes, and I still have to write about 30 lines of code to get it to work the way I want.
```php
// Customize the entry footer.
add_action( 'alcatraz_before', function() {
	remove_action( 'alcatraz_entry_footer_inside', 'alcatraz_output_default_entry_footer' );
	add_action( 'alcatraz_entry_footer_inside', 'clayton_gerard_entry_footer' );
	/**
	 * Customize the entry footer.
	 */
	function clayton_gerard_entry_footer( $post_id = 0 ) {

		if ( ! $post_id ) {
			$post_id = get_the_ID();
		}

		$footer_taxonomies = array(
			'categories' => __( 'Posted in: ', 'clayton-gerard' )
		);

		foreach ( $footer_taxonomies as $footer_taxonomy => $label ) {
			alcatraz_the_taxonomy_term_list( $post_id, $footer_taxonomy, $label, ', ' );
		}

		if ( ! is_single() && ! post_password_required() && ( comments_open() || get_comments_number() ) ) {

			echo '<span class="comments-link">';
			comments_popup_link( esc_html__( 'Leave a Comment', 'alcatraz' ), esc_html__( '1 Comment', 'alcatraz' ), esc_html__( '% Comments', 'alcatraz' ) );
			echo '</span>';
		}

		alcatraz_the_edit_post_link( $post_id );
	}
} );
```
And this even isn't working. :( I'm concerned that this is too complicated.